wordpress-mobile/gutenberg-mobile
# Fix image preview with small image size selected

This PR fixes an issue that happens when the user selects a thumbnail (or any smaller image size), and then press on it so see it full screen (this happens on self-hosted sites).

`Gutenberg` side PR: https://github.com/WordPress/gutenberg/pull/19247
`WPiOS` side PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/13145

Expected: The user sees the full sized image.
Result: The user sees the smaller, low quality version of the image.

Also on this PR:
- Sending both the current image url (sized) and original url (full size). This is not currently used for image preview, but will be used for Image Editing. Since this is iOS only (so far), the Android version of it will still receive only one parameter, but still fixing the original issue. 

**Note:** I'm planning/thinking about renaming the `requestImageFullscreenPreview` method action to something more generic, like `requestImageAction`, with an action parameter that could be `preview`, `edit`, etc... so we can reuse it for various future actions.

![small_preview](https://user-images.githubusercontent.com/9772967/71173968-536dcc00-2264-11ea-9366-9f8200030f20.gif)

To test:
- Open Gutenberg on WPiOS, on a self-hosted site.
- Add a new image.
- Change its size to thumb.
- Tap on the image to show the image preview.
- Check that the image shown is the original full size image.

PR submission checklist:
